src/pages/gen2/start/account-setup/index.mdx

```diff
@@ -9,105 +9,97 @@ export function getStaticProps(context) {
-This guide will walk you through how to set up your AWS account by creating a local profile and by completing a one-time bootstrapping process to perform CI/CD deployments for your Amplify (Gen 2) applications.
+{/* "...configure an AWS profile locally for use with Amplify" -> `amplify configure profile` */}
+This guide will walk you through how to set up your AWS account and configure an AWS profile locally for use with Amplify. Before getting started it is important to know there are a few options for authenticating with AWS. Using [AWS Identity and Access Management (IAM)](https://aws.amazon.com/iam/), you can configure either:
+
+- permanent credentials with an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) through the use of access keys
+- temporary credentials with an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)
+- temporary credentials with [IAM Identity Center](https://docs.aws.amazon.com/singlesignon/latest/userguide/what-is.html), which is formerly known as AWS Single Sign-On (SSO)
 
-## Configure local machine
+Temporary credentials are an alternative to their permanent counterpart and enable you to define permissions for the _session_. Sessions are created when you [_assume_ an IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use.html) or sign in using AWS IAM Identity Center. These sessions come with an additional "session token" that is used to validate the temporary credentials and must be included on requests to AWS. As you are working locally, this will be presented as an additional environment variable.
 
-You will need to configure your local machine to connect to your AWS account before you can set up the [Amplify Sandbox environment](/gen2/deploy-and-host/sandbox-environments/setup/).
+> You can use temporary security credentials to make programmatic requests for AWS resources using the AWS CLI or AWS API (using the AWS SDKs). The temporary credentials provide the same permissions as long-term security credentials, such as IAM user credentials. However, there are a few differences:...
+>
+> [AWS Identity and Access Management documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html)
 
-### Setup a local profile
+In general, though, credentials are typically resolved through the use of [AWS profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-using-profiles). Profiles can contain permanent credentials or SSO metadata, and can be set for use with Amplify by using the same techniques as the AWS CLI:
+
+- with the `--profile` flag
+- with the `AWS_PROFILE` environment variable
 
 <Callout info>
 
-**Note**: If you already have an AWS profile with credentials on your machine, you can skip this step.
+**Note**: If you already have an AWS account and profile configured locally, you can skip this guide and specify a profile using one of the methods noted above.
```

Comment:
Keep this callout above all the new information

Reply:
this callout mentions "the methods noted above". how do you feel about changing that or removing it entirely?

Reply:
I don't feel that strongly about it. It just seems like the text above is a lot to wade through only to realize you don't need any of it if you already have credentials

Reply:
fair callout, lifted the callout to the top